specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/machineLearningServices.json

```diff
@@ -2026,6 +2335,23 @@
       "description": "Category of the workspace connection.",
       "required": false,
       "x-ms-parameter-location": "method"
+    },
+    "RuleName": {
+      "in": "path",
+      "name": "ruleName",
+      "description": "Name of the workspace managed network outbound rule",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "pattern": "^[a-zA-Z0-9][a-zA-Z0-9_-]{2,32}$"
+    },
+    "WorkspaceForceToPurgeParameter": {
```

Comment:
for slightly simpler names, I'd suggest to call it just `ForceParameter` or `ForcePurgeParameter`

Reply:
will keep it as this as this is already used by SDK and release.